TCAT/AppDelegate.swift

```diff
@@ -82,6 +86,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.window!.rootViewController = navigationController
         self.window?.makeKeyAndVisible()
         
+        // Check app version
+        if let version = userDefaults.value(forKey: Constants.UserDefaults.version) as? String {
+            if version != Constants.App.version {
+                showWhatsNew(items: [])
```

Comment:
My understanding of this is that if the version number is different, the WhatsNew feature will be presented, but that didn't happen. We wouldn't want that for this upcoming smaller release, but we will in the future. 

I'm not sure why that didn't work currently, but I do know WhatsNew contains a built in system to manage the version. The initializer for the WhatsNewViewController is optional because it check these booleans to see if it should be presented or not.

We implemented this class in Brella, which updates the User Defaults, and skips patch updates.
```
class MyUserSettingsDatabase: WhatsNewVersionStore {
    /// Returns true if What's New should be shown
    /// Patch versions (e.g. X.x.0) don't count.
    func has(version: WhatsNew.Version) -> Bool {
        let isVersionPatch = version.patch > 0
        let isNotNewVersion = App.version == data.value(forKey: Keys.version) as! String
        return isVersionPatch || isNotNewVersion
    }
    
    func set(version: WhatsNew.Version) {
        data.set(App.version, forKey: Keys.version)
        return
    }
}
```

Let's talk more in person!

Reply:
`
    let version = WhatsNew.Version(App.Version)
    let versionStore = MyUserSettingsDatabase()
    if versionStore.has(version) {
        presentCard()
    }
`